### PR TITLE
add contributing ready for review step

### PR DESCRIPTION
**Pull request by mkasztelnik**
new step (nr 6) added to contributing section

/cc @dice-cyfronet/atmo-dev-team please take a look

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,4 +109,6 @@ rake atmosphere:install:migrations
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
-5. Create new pull request
\ No newline at end of file
+5. Create new pull request
+6. When feature is ready add "ready for revieew" label and mention atmosphere
```
Comment (paoolo) [inline README.md]: Please change from ``ready for revieew`` to ``ready for review``.

**Comment by mkasztelnik** [comment]
@paoolo - corrected

**Comment by paoolo** [comment]
:+1:

**Comment by Nuanda** [comment]
:+1:

**Pull request merged**